appinventor/appengine/src/com/google/appinventor/client/explorer/youngandroid/ProjectToolbar.java

```diff
@@ -9,19 +9,20 @@
 import com.google.appinventor.client.Ode;
 import com.google.appinventor.client.boxes.ProjectListBox;
 import com.google.appinventor.client.widgets.Toolbar;
-import com.google.appinventor.client.widgets.ToolbarItem;
 import com.google.gwt.core.client.GWT;
 import com.google.gwt.uibinder.client.UiBinder;
 import com.google.gwt.uibinder.client.UiField;
+import java.util.logging.Logger;
 
+import com.google.gwt.user.client.ui.Label;
 
 /**
  * The project toolbar houses command buttons in the Young Android Project tab.
  *
  */
 public class ProjectToolbar extends Toolbar {
+  private static final Logger LOG = Logger.getLogger(ProjectToolbar.class.getName());
   interface ProjectToolbarUiBinder extends UiBinder<Toolbar, ProjectToolbar> {}
-
   private static final ProjectToolbar.ProjectToolbarUiBinder UI_BINDER =
```

Comment:
I don't recall if we have a custom project toolbar at this point but if so you may want to move the initialization of this into the bindUI function so that it won't bundle both the classic and neo UIs into the same JS file.